### PR TITLE
Master mdl 71613 59

### PR DESCRIPTION
**Pull request by marcghaly**
- Fixed code issues in https://github.com/catalyst/moodle-MDL-70329/pull/130

- New PR with proper branching

### CODE REVIEW

File: question/bank/viewcreator/classes/columns.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+defined('MOODLE_INTERNAL') || die();
+
+use core_question\local\bank\column_plugin_base;
+
+/**
+ * Class columns is the entrypoint for the columns.
+ *
+ * @package    qbank_viewcreator
+ * @copyright  2021 Catalyst IT Canada Pty Ltd
```
Comment (safatshahin) [inline question/bank/viewcreator/classes/columns.php:36]: Please use Catalyst IT Australia in all copyright as matt recommended

**Review by safatshahin — COMMENTED** [review]
Hi @marcghaly 
All good with the code, please double check the copyright information in some files and then we are good to merge.

File: question/bank/viewcreator/classes/columns.php (hunk 1 above)
Comment (marcghaly) [reply question/bank/viewcreator/classes/columns.php]: Hi @safatshahin :

- Copyright in `version.php`, `columns.php` and `provider.php` set to Australia
- Phpunit tests ok
- Codechecker run and ok

File: question/bank/viewcreator/templates/creator_display.mustache [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        "displaydata": [
+          {
+                "creator": "Marc Ghaly",
+                "date": "2 June 2021, 5:32 PM",
+          }
+        ]
+    }
+}}
+
+
```
Comment (guillogo) [inline question/bank/viewcreator/templates/creator_display.mustache:27]: can you please remove this space.

File: question/bank/viewcreator/templates/creator_display.mustache [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+<span>
```
Comment (guillogo) [inline question/bank/viewcreator/templates/creator_display.mustache:28]: I'd add a class to spam tag in order to add style for future theme developers.

File: question/bank/viewcreator/templates/modifier_display.mustache [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        "displaydata": [
+          {
+                "modifier": "Marc Ghaly",
+                "date": "2 June 2021, 5:32 PM",
+          }
+        ]
+    }
+}}
+
+<span>
```
Comment (guillogo) [inline question/bank/viewcreator/templates/modifier_display.mustache:27]: I'd add a class to spam tag in order to add style for future theme developers.

**Review by guillogo — CHANGES_REQUESTED** [review]
Hi @marcghaly 

Review done. Added 2 comments (easy changes). 

@safatshahin what do you think about having 2 mustaches that do the same thing? thoughts?

Thank you.

File: question/bank/viewcreator/templates/creator_display.mustache (hunk 2 above)
Comment (marcghaly) [reply question/bank/viewcreator/templates/creator_display.mustache:27]: Space removed

File: question/bank/viewcreator/templates/creator_display.mustache (hunk 3 above)
Comment (marcghaly) [reply question/bank/viewcreator/templates/creator_display.mustache:28]: `class="creatorclass"` added

File: question/bank/viewcreator/templates/modifier_display.mustache (hunk 4 above)
Comment (marcghaly) [reply question/bank/viewcreator/templates/modifier_display.mustache:27]: `class="modifierclass"` added

**Pull request merged**